controllers/roverController.ts

```diff
@@ -0,0 +1,16 @@
+import nasaClient from "../clients/nasaClient"
+import Rover from "../models/roverData"
+
+export const RoverController = {
+    Index: async (req: any, res: any) => {
+        try {
+            const roverData: Rover[] | void = await nasaClient.getRoverList()
```

Comment:
I don't think `void` should be used here? `void` is a keyword used for a function that has no return value (for the absence of information you should be using `null` or `undefined`)

Reply:
It seems that the catch block in `getRoverList()` would return `void` since it only sends the error to the console rather than returning it. When replacing with `null` it highlights that it should be void, but `null` was my first instinct to use here

Reply:
Does `undefined` also not work?

Reply:
Oh, it does!